### DISCUSSION TITLE
Load functionality

### DISCUSSION THREAD

**Discussion by ibdul** [danielcopper/wezterm-session-manager#13] [Show and tell]
Hi everyone.

I made the load functionality and I've been using it for about a day and decided it works fine and I should share and if approved, make a PR. 

first, some changes to the current setup. 
Since the session manager practically works with the workspaces, I decided to tie them up in my implementation. This ensures that We can have "persistent" wezterm workspaces.
How this works (Summary)
when you create a new workspace, it creates a session with same name (with the file and all, that is where persistence comes in). 

![image](https://github.com/danielcopper/wezterm-session-manager/assets/26022162/0140955a-4cf4-4459-9416-e9be747f13d2)
When you try to load, all it does is scans the sessions directory to find the saved sessions. you choose one, it loads the configurations, and we create a wezterm workspace with same name.

it all comes together to give the illusion that your workspaces where persisted.

**Comment by akthe-at** [comment]
Ibdul, I think this sounds great and would save me a bunch of keystrokes
